tests/_support/AcceptanceTester.php

```diff
@@ -141,4 +151,41 @@ public function searchFor($query, $delay = 0, $element = '#search_input', $butto
     $I->click($button);
   }
 
+  /**
+   * Override waitFor* methods to have a common default timeout
+   */
+  public function waitForElement($element, $timeout = 10) {
+    return $this->_waitForElement($element, $this->getDefaultTimeout($timeout));
+  }
+
+  public function waitForElementChange($element, \Closure $callback, $timeout = 30) {
+    return $this->_waitForElementChange($element, $callback, $this->getDefaultTimeout($timeout));
+  }
+
+  public function waitForElementClickable($element, $timeout = 10) {
+    return $this->_waitForElementClickable($element, $this->getDefaultTimeout($timeout));
+  }
+
+  public function waitForElementNotVisible($element, $timeout = 10) {
+    return $this->_waitForElementNotVisible($element, $this->getDefaultTimeout($timeout));
+  }
+
+  public function waitForElementVisible($element, $timeout = 10) {
+    return $this->_waitForElementVisible($element, $this->getDefaultTimeout($timeout));
+  }
+
+  public function waitForJS($script, $timeout = 5) {
+    return $this->_waitForJS($script, $this->getDefaultTimeout($timeout));
+  }
+
+  public function waitForText($text, $timeout = 10, $selector = null) {
+    return $this->_waitForText($text, $this->getDefaultTimeout($timeout), $selector);
+  }
+
+  private function getDefaultTimeout($timeout) {
+    return (int)getenv('WAIT_TIMEOUT')
+      ?: self::DEFAULT_WAIT_TIMEOUT
+      ?: $timeout;
+  }
```

Comment:
So looking at [line 157](https://github.com/mailpoet/mailpoet/pull/1754/files#diff-aa3129a09f27f044b98b567399794bd6R157) `public function waitForElement($element, $timeout = 10) {` this means that explicit timeout would be always ignored, right?

Couldn't we change the defaults in method headers to `$timeout = null` and put `$timeout` to the first position? Then the priorities would be more readable:
1. Explicit `waitForElement($element, 20)`.
2. Environment's `WAIT_TIMEOUT`.
3. The `DEFAULT_WAIT_TIMEOUT` constant.

What do you think? Of course we want to avoid 1. but it would still work as expected. Or maybe what we want is having the ability to override even explicit timeouts with the EVN variable? Then the constant with default timeout should be last and `$timeout` in the middle.

Reply:
Explicit timeouts will still work if `WAIT_TIMEOUT` and `DEFAULT_WAIT_TIMEOUT` are not set, like it is now. If we make the argument `$timeout = null` everywhere, we will lose the default timeouts of Codeception because they're set as default function arguments. 

Yes, we do want to override both explicit and default timeouts globally because we only need to increase them.

Reply:
You're right, the constant is unnecessary, I've removed it. If we ever need to set the default timeout for all environments we can always add it later.